src/EnergyPlus/WindowManager.cc

```diff
@@ -4534,7 +4515,7 @@ namespace Window {
                 fcon[i - 1] = wmgas.con.c0 + wmgas.con.c1 * tmean + wmgas.con.c2 * tmean_2;
                 fvis[i - 1] = wmgas.vis.c0 + wmgas.vis.c1 * tmean + wmgas.vis.c2 * tmean_2;
                 fcp[i - 1] = wmgas.cp.c0 + wmgas.cp.c1 * tmean + wmgas.cp.c2 * tmean_2;
-                fdens[i - 1] = pres * wmgas.wght / (gaslaw * tmean);
+                // fdens[i - 1] = pres * wmgas.wght / (gaslaw * tmean);
```

Comment:
[src/EnergyPlus/WindowManager.cc:4537]:(style),[unreadVariable],Variable 'fdens[i-1]' is assigned a value that is never used.